data/35e/wizards_of_the_coast/supplement/monster_manual_2/mm2_templates.lst

```diff
@@ -12,7 +12,7 @@ Captured One																	VISIBLE:YES
 ###TODO: Death Knight: where this DR:15/Magic and Bludgeoning came from?
 Death Knight								HITDIE:12|CLASS.TYPE=Monster					VISIBLE:YES		DR:5/Magic					LEVELADJUSTMENT:5																																																									SOURCEPAGE:MM2	DEFINE:DeathKnightSR|0			DEFINESTAT:NONSTAT|CON	PREALIGN:LE,NE,CE						PRELEVEL:MIN=6																														PRERACE:1,RACETYPE=Humanoid,RACETYPE=Giant,RACETYPE=Monstrous Humanoid																																									BONUS:COMBAT|AC|5|TYPE=NaturalArmor		BONUS:MISC|CR|3																																																		BONUS:STAT|STR|4	BONUS:STAT|WIS|2	BONUS:STAT|CHA|2										BONUS:VAR|DeathKnightSR|20+(TL-10)	ABILITY:Special Ability|AUTOMATIC|Death Knight ~ Constitution Damage|Death Knight ~ Abyssal Blast|Death Knight ~ Fear Aura|Death Knight ~ Undead Followers	ABILITY:Special Ability|AUTOMATIC|Immunity To Cold|Immunity To Electricity|Immunity To Polymorph	ABILITY:Special Ability|AUTOMATIC|Death Knight ~ Immunity to Turning																SR:DeathKnightSR						RACETYPE:Undead																													NATURALATTACKS:Touch,Weapon.Natural.Finesseable.Melee.Bludgeoning,*1,1d8
-Golem (Half Golem)			OUTPUTNAME:Half Golem										VISIBLE:YES											TEMPLATE:CHOOSE:Golem (Half/Clay)|Golem (Half/Flesh)|Golem (Half/Iron)|Golem (Half/Stone)																																										SOURCEPAGE:MM2																						PREMULT:1,[PRERACE:1,RACETYPE=Animal],[PRERACE:1,RACETYPE=Giant],[PRERACE:1,RACETYPE=Humanoid],[PRERACE:1,RACETYPE=Magical Beast],[PRERACE:1,RACETYPE=Monstrous Humanoid]																																														BONUS:CHECKS|Fortitude|2|TYPE=Racial								BONUS:MISC|CR|3																																																		BONUS:STAT|DEX|-2	BONUS:STAT|CON|4	BONUS:STAT|INT|-6	BONUS:STAT|CHA|-6													ABILITY:Special Ability|AUTOMATIC|Golem Shared ~ Immunity To Magic
+Golem (Half Golem)			OUTPUTNAME:Half Golem										VISIBLE:YES											TEMPLATE:CHOOSE:Golem (Half/Clay)|Golem (Half/Flesh)|Golem (Half/Iron)|Golem (Half/Stone)																																										SOURCEPAGE:MM2																						PREMULT:1,[PRERACE:1,RACETYPE=Animal],[PRERACE:1,RACETYPE=Giant],[PRERACE:1,RACETYPE=Humanoid],[PRERACE:1,RACETYPE=Magical Beast],[PRERACE:1,RACETYPE=Monstrous Humanoid]																																														BONUS:SAVE|Fortitude|2|TYPE=Racial								BONUS:MISC|CR|3																																																		BONUS:STAT|DEX|-2	BONUS:STAT|CON|4	BONUS:STAT|INT|-6	BONUS:STAT|CHA|-6													ABILITY:Special Ability|AUTOMATIC|Golem Shared ~ Immunity To Magic
```

Comment:
This would break PCGen version 6.04.01. BONUS:SAVE did not become valid until the 6.06 development cycle. This type of change only belongs in the master branch. Revert this back please.